manifest-v2.json

```diff
@@ -191,16 +191,6 @@
       "useCaseName": "Data processing"
     }
   ],
-  "aot.dotnet7 (provided.al2)": [
```

Comment:
Does the integ tests order need to be updated for the SAM CLI integration tests?

When I run `sam init`, I see aot.dotnet7 as the first choice:

```
Which runtime would you like to use?
	1 - aot.dotnet7 (provided.al2)
	2 - dotnet8
	3 - dotnet6
	4 - go (provided.al2)
	5 - go (provided.al2023)
	6 - graalvm.java11 (provided.al2)
	7 - graalvm.java17 (provided.al2)
	8 - java21
	9 - java17
```

Reply:
we will merge this first then check in SAM CLI develop branch